package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Attempt at simple reproducible execution example (from the dependent's end)",
   "main": "index.js",
   "scripts": {
+    "cucumber": "cucumber-js --require node_modules/@plocket/cucumber-exec/index.js --publish-quiet",
```

Comment:
For this to work, needed to comment out this line in `index.js` which refers to a file that doesn't exist:

```js
require('./world');
```